x-pack/plugin/searchable-snapshots/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/sort_segments_on_timestamp.yml

```diff
@@ -0,0 +1,88 @@
+---
+setup:
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              "@timestamp":
+                type: date
+          settings:
+            number_of_shards: 1
+            number_of_replicas: 0
```

Comment:
Can we rely on the default number of replicas?

Reply:
makes sense, shall I also remove setting the number of primary shards?

Reply:
No, I think the test can only work with `number_of_shards: 1` and as such it should be skipped for non default shard number. See `features: default_shards`